db/channel_cache_test.go

```diff
@@ -431,6 +433,208 @@ func TestChannelCacheRemove(t *testing.T) {
 	goassert.True(t, err == nil)
 }
 
+func TestChannelCacheStats(t *testing.T) {
+
+	defer base.SetUpTestLogging(base.LevelInfo, base.KeyCache)()
+
+	context := testBucketContext()
+	defer context.Close()
+	defer base.DecrNumOpenBuckets(context.Bucket.GetName())
```

Comment:
Any reason why we can't move `base.DecrNumOpenBuckets(context.Bucket.GetName())` into `context.Close()`

Reply:
Oh, I thought it was some sort of testing context, nevermind.